app/containers/ItunesContainer/reducer.js

```diff
@@ -0,0 +1,49 @@
+/*
+ *
+ * itunesContainer reducer
+ *
+ */
+
+import produce from 'immer';
+import { createActions } from 'reduxsauce';
+
+export const initialState = {
+  searchTerm: null,
+  gridData: {},
+  searchError: null
+};
```

Comment:
can w give a better naming to these ?

trackName, tracksData, tracksError

Reply:
Yes, updated the change accordingly